django/black.cfg

```diff
@@ -0,0 +1,3 @@
+[unittest]
```

Comment:
Did you mean to commit this file? If so it should probably have a more descriptive name.